src/lib.rs

```diff
@@ -96,6 +96,71 @@
 #![cfg_attr(nightly_docs, deny(broken_intra_doc_links))]
 #![cfg_attr(nightly_docs, feature(doc_cfg))]
 
+/// Defines a mark to be used with [`hit!`] and [`chk!`].
+///
+/// # Example
+///
+/// ```
+/// #[cfg(test)]
+/// mod tests {
+///    cov_mark::def!(no_op);
+/// }
+/// ```
+#[macro_export]
+macro_rules! def {
+    ($ident:ident) => {
+        $crate::__cov_mark_private_create_mark! { static $ident }
```

Comment:
I would just adjust and rename the `__cov_mark_private_create_mark` macro instead of creating a new one.

Reply:
I removed the `static` and changed name (`define`?), not sure if you had in mind something else.

Reply:
Yeah, seems good to me.